aws-events-connection/aws-events-connection.json

```diff
@@ -0,0 +1,287 @@
+        "HeaderParameters": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Parameter"
+          }
+        },
+        "QueryStringParameters": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Parameter"
+          }
+        },
+        "BodyParameters": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Parameter"
+          }
+        }
+      },
+      "additionalProperties": false
+    },
+    "Parameter": {
+      "type": "object",
+      "properties": {
+        "Key": {
+          "type": "string"
+        },
+        "Value": {
+          "type": "string"
+        },
+        "IsValueSecret": {
+          "type": "boolean",
+          "default": true
+        }
+      },
+      "required": [
+        "Key",
+        "Value"
+      ],
+      "additionalProperties": false
+    }
+  },
+  "properties": {
+    "Name": {
+      "description": "Name of the connection.",
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 64
+    },
+    "Arn": {
+      "description": "The arn of the connection resource.",
+      "type": "string"
+    },
+    "SecretArn": {
+      "description": "The arn of the secrets manager secret created in the customer account.",
+      "type": "string"
+    },
+    "Description": {
+      "description": "Description of the connection.",
+      "type": "string",
+      "maxLength": 512
+    },
+    "AuthorizationType": {
+      "type": "string",
+      "enum": [
+        "API_KEY",
+        "BASIC",
+        "OAUTH_CLIENT_CREDENTIALS"
+      ]
+    },
+    "AuthParameters": {
+      "type": "object",
+      "minProperties": 1,
+      "maxProperties": 2,
+      "properties": {
+        "ApiKeyAuthParameters": {
+          "$ref": "#/definitions/ApiKeyAuthParameters"
+        },
+        "BasicAuthParameters": {
+          "$ref": "#/definitions/BasicAuthParameters"
+        },
+        "OAuthParameters": {
+          "$ref": "#/definitions/OAuthParameters"
+        },
+        "InvocationHttpParameters": {
+          "$ref": "#/definitions/ConnectionHttpParameters"
+        }
+      },
+      "oneOf": [
+        {
+          "required": [
+            "BasicAuthParameters"
+          ]
+        },
+        {
+          "required": [
+            "OAuthParameters"
+          ]
+        },
+        {
+          "required": [
+            "ApiKeyAuthParameters"
+          ]
+        }
+      ],
+      "additionalProperties": false
+    }
+  },
+  "additionalProperties": false,
+  "required": [
+    "AuthorizationType",
+    "AuthParameters"
+  ],
+  "createOnlyProperties": [
+    "/properties/Name"
+  ],
+  "readOnlyProperties": [
+    "/properties/Arn",
+    "/properties/SecretArn"
+  ],
+  "writeOnlyProperties": [
+    "/definitions/BasicAuthParameters/Password",
+    "/definitions/ApiKeyAuthParameters/ApiKeyValue",
+    "/definitions/OAuthParameters/ClientParameters/ClientSecret"
+  ],
```

Comment:
This is the only public resource schema pointing towards `/definitions/` instead of `/properties/`

Please double check this is working as expected and switch them if not